k8s/agent-scaler/agent-scaler.sh

```diff
@@ -4,7 +4,7 @@ batch_size=${BATCH_SIZE:-1}
 max_jobs=${MAX_JOBS:-10}
 check_delay=${CHECK_DELAY:-10}
 
-namespace=$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace)
+namespace=${NAMESPACE:-joshua}
```

Comment:
Do we want to use the "Joshua" namespace as kubernetes namespace?
i.e. Joshua has a concept of namespaces, which isolates ensembles inside the coordinating cluster.

I personally think retrieving the "current" kubernetes namespace through /var/run is still a better approach here.

Reply:
change made.